Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/TextLogFileAcceptanceTest.cs

```diff
@@ -383,7 +383,7 @@ public void TestReadAll1()
 
 				file.Dispose();
 
-				file.GetValue(LogFileProperties.StartTimestamp).Should().Be(new DateTime(2015, 10, 7, 19, 50, 58, 982));
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.